src/panels/lovelace/cards/hui-picture-entity-card.js

```diff
@@ -122,9 +138,19 @@ class HuiPictureEntityCard extends LocalizeMixin(PolymerElement) {
     }
   }
 
+  _computeHideInfo(config) {
+    // By default we will show it, so === undefined should be true.
+    return config.show_info === false;
```

Comment:
can we document this in PR description. It's for someone to keep docs updated if it's not there